.env.docker

```
@@ -45,9 +48,15 @@ DC_MEDUSA_COOKIE_SECURE=
 DC_MEDUSA_COOKIE_SAME_SITE=
 DC_MEDUSA_ADMIN_DISABLED_FOR_BACKEND_BUILD=0
 DC_STOREFRONT_URL=http://localhost:3001
-DC_STORE_CORS="http://localhost:3000,http://127.0.0.1:3000,http://localhost:3001,http://127.0.0.1:3001,http://localhost:8000,http://127.0.0.1:8000,http://localhost:8080,http://127.0.0.1:8080,https://n1.medusa.localhost,http://localhost:9004,https://docs.medusajs.com"
-DC_ADMIN_CORS="http://localhost:5173,https://admin.medusa.localhost,http://localhost:9000,https://docs.medusajs.com"
-DC_AUTH_CORS="http://localhost:5173,https://admin.medusa.localhost,http://localhost:9000,https://docs.medusajs.com"
+DC_STORE_NAME="NewEngine Store"
+DC_STORE_CORS="http://localhost:3000,http://127.0.0.1:3000,http://localhost:3001,http://127.0.0.1:3001,http://localhost:8000,http://127.0.0.1:8000,http://localhost:8080,http://127.0.0.1:8080,https://n1.medusa.localhost,http://localhost:9004"
+DC_ADMIN_CORS="http://localhost:5173,https://admin.medusa.localhost,http://localhost:9000"
+DC_AUTH_CORS="http://localhost:5173,https://admin.medusa.localhost,http://localhost:9000"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)